cmd/tegola/cmd/root.go

```diff
@@ -16,13 +16,25 @@ import (
 var (
 	logLevel   string
 	configFile string
+	logger     string
 	// parsed config
 	conf config.Config
 
 	// require cache
 	RequireCache bool
 )
 
+func validateSupportedLoggers(logger string) error {
+	switch logger {
+	case log.STANDARD:
+		return nil
+	case log.ZAP:
+		return nil
+	default:
+		return fmt.Errorf("invalid logger %s", logger)
+	}
+}
+
 func getLogLevelFromString(level string) (log.Level, error) {
 	switch level {
 	case "TRACE":
```

Comment:
Do we have some consts we can key off of for these log levels?

Reply:
Not sure if I understand your right here. Log levels are numbers ranging from -2 to 3 e.g. `log.TRACE ` equals -2 and we have to derive this from the users input string.
We could add additional consts like `TRACE_STRING` for that switch statement I guess?